simpledotorg/simple-server
# Cancel outdated experiments and create new experiments for India

**Story card:** [sc-11186](https://app.shortcut.com/simpledotorg/story/11186/bsnl-bug-re-enable-sms-experiments-in-india)

## Because

Migrating to new deployment environment in India had caused pending notifications to cause a thundering herd, leading to blacklisting from BSNL.

## This addresses

Cancels existing experiments and their notifications, evicts patients in those experiments and creates new experiments for month of Nov & Dec.

## Test instructions

Uncomment country check and run `rake data:migrate`. Expect 4 experiments to be created (2 current & 2 stale for months of Nov/Dec)
